aten/src/ATen/CMakeLists.txt

```diff
@@ -156,6 +154,15 @@ file(GLOB native_transformers_hip_cpp "native/transformers/hip/*.cpp")
 file(GLOB native_quantized_cudnn_hip_cpp "native/quantized/cudnn/hip/*.cpp")
 file(GLOB native_utils_cpp "native/utils/*.cpp")
 
+# Add flash attention sources if condition
+if(USE_FLASH_ATTENTION)
+  file(GLOB native_transformers_cuda_cu "native/transformers/cuda/*.cu" "native/transformers/cuda/flash_attn/*.cu")
+  file(GLOB native_transformers_cuda_cpp "native/transformers/cuda/*.cpp" "native/transformers/cuda/flash_attn/*.cpp")
+else()
+  file(GLOB native_transformers_cuda_cu "native/transformers/cuda/*.cu")
+  file(GLOB native_transformers_cuda_cpp "native/transformers/cuda/*.cpp")
+endif()
```

Comment:
To follow the same pattern, just create a new glob var and then append it conditionally to the list